charts/tfy-llm-gateway/values.yaml

```diff
@@ -274,3 +274,67 @@ oidc:
   enabled: false
   ## @param oidc.config OIDC configuration
   config: {}
+## Redis dependency configuration
+redis:
+  ## @param redis.enabled Enable Redis dependency
+  enabled: false
```

Comment:
add a global var called redis enabled and ONLY if that is enabled - redis has to be marked true - defaults to false

also plz make sure that we inject the url of redis by default also